owtf/webapp/src/containers/SignupPage/index.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * SignupPage.
+ * Handles the signup of the new user
+ * Redirects to the email confirmation page after input validation
+ */
+
+import React from "react";
+import {
+  Pane,
+  Heading,
+  Button,
+  Text,
+  Link,
+  TextInputField
+} from "evergreen-ui";
+import {
+  GoogleLoginButton,
+  GithubLoginButton,
+  TwitterLoginButton
+} from "react-social-login-buttons";
+const OAuthBtnStyle = { width: "60%", marginLeft: "20%" };
+
+export class SignupPage extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      email: "",
+      password: "",
+      confirm_password: "",
+      username: ""
+    };
+  }
+
+  render() {
+    return (
+      <Pane marginY={40}>
+        <Pane justifyContent="center" width="35%" elevation={1} margin="auto">
+          <Heading
+            size="700"
+            textAlign="center"
+            marginBottom={20}
+            paddingTop={30}
+          >
+            Signup
+          </Heading>
+          <TextInputField
+            placeholder="Username"
+            width="60%"
+            marginLeft="20%"
+            marginBottom={20}
+            value={this.state.username}
+            onChange={e => this.setState({ username: e.target.value })}
+          />
+          <TextInputField
+            placeholder="Email"
+            width="60%"
+            marginLeft="20%"
+            marginBottom={20}
+            value={this.state.email}
+            onChange={e => this.setState({ email: e.target.value })}
+          />
+          <TextInputField
+            placeholder="Password"
+            width="60%"
+            marginLeft="20%"
+            marginBottom={20}
+            value={this.state.password}
+            onChange={e => this.setState({ password: e.target.value })}
+          />
+          <TextInputField
+            placeholder="Confirm Password"
+            width="60%"
+            marginLeft="20%"
+            marginBottom={20}
+            value={this.state.confirm_password}
+            onChange={e => this.setState({ confirm_password: e.target.value })}
+          />
+          <Button
```

Comment:
If time permits, can you add a functionality that enables the submit buttons only after the form is filled with valid responses.

Reply:
This PR only defines the basic UI structure of the different pages. Noted it and will add proper form validation and will make button active/disabled suitably in the upcoming PR.